pnpm-lock.yaml

```diff
@@ -16680,7 +16598,7 @@ snapshots:
       '@babel/core': 7.25.2
       '@babel/helper-module-transforms': 7.25.7(@babel/core@7.25.2)
       '@babel/helper-plugin-utils': 7.25.7
-      '@babel/helper-simple-access': 7.24.7
+      '@babel/helper-simple-access': 7.25.7
```

Comment:
not sure the changes to pnpm-lock are related to those changes, maybe good to reset and rerun pnpm install to be sure

Reply:
@timotheeguerin Reset and run removed the lock file changes.